src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -56085,6 +56444,7 @@ void M0<T>(T x0, T y0)
         M1(z0) = y0;
         z0.ToString();
         z0?.ToString();
+        M1(z0).ToString();
```

Comment:
>M1(z0).ToString(); [](start = 8, length = 18)

The line ```z0.ToString()``` above actually changes the flow sate of ```z0```, consider placing this line above it, so that both ```M1(z0)```` calls are based on the same state.